internal/backends/collection.go

```diff
@@ -82,8 +82,8 @@ type QueryParams struct {
 	// TODO https://github.com/FerretDB/FerretDB/issues/3235
 	Filter        *types.Document
 	Sort          *SortField
-	Limit         int64  // if 0 no limit pushdown is applied
-	OnlyRecordIDs bool   // TODO https://github.com/FerretDB/FerretDB/issues/3490
+	Limit         int64 // if 0 no limit pushdown is applied
```

Comment:
While we are on it…

```suggestion
	Limit         int64
```

https://twitter.com/carterwickstrom/status/1014165500056596481